src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp

```diff
@@ -61,6 +61,16 @@ class SubgraphBaseTest : public CommonTestUtils::TestsCommon {
     virtual std::vector<ov::Tensor> get_plugin_outputs();
 };
 
+inline std::vector<InputShape> dynamic_shapes_to_test_representation(const std::vector<ov::PartialShape>& shapes) {
```

Comment:
Is there a example of usage?

Reply:
Of course, please have a look [here](https://github.com/a-sidorova/openvino/blob/6cb1bae48037cb484a149cf02c3483e8d35417e0/src/tests/functional/plugin/shared/src/snippets/two_inputs_and_outputs.cpp#L31) or [here](https://github.com/a-sidorova/openvino/blob/6cb1bae48037cb484a149cf02c3483e8d35417e0/src/tests/functional/plugin/shared/src/snippets/convert.cpp#L90).